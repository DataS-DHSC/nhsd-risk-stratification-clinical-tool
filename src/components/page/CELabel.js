import React from 'react';

const CELabel = () => (
  <>
    <div className="nhsuk-grid-row">
      <div className="nhsuk-grid-column-two-thirds">
        <h1>Redacted for open source publication reasons</h1>
      </div>
    </div>
  </>
);

export default CELabel;
