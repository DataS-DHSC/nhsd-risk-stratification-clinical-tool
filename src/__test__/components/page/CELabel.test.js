import React from 'react';
import { render } from '@testing-library/react';
import CELabel from '../../../components/page/CELabel';

const setup = () => render(<CELabel />);

describe('CELabel', () => {
  it('should match snapshot', () => {
    const container = setup();

    expect(container.asFragment()).toMatchSnapshot();
  });
});
