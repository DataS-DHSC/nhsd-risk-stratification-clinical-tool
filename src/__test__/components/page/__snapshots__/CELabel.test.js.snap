// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CELabel should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="nhsuk-grid-row"
  >
    <div
      class="nhsuk-grid-column-two-thirds"
    >
      <h1>
        Redacted for open source publication reasons
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
